crates/solidity/testing/snapshots/cst_output/ContractDefinition/zero_length_input/generated/0.4.11.yml

```diff
@@ -7,4 +7,5 @@ Errors: # 1 total
     Error: Expected ContractKeyword.
        ─[crates/solidity/testing/snapshots/cst_output/ContractDefinition/zero_length_input/input.sol:0:0]
 
-Tree: null
+Tree:
+  - SKIPPED (Token): [] # 0..0
```

Comment:
Nothing is skipped in this case. Is the zero-length token intentional?

Reply:
Yes, because I need some token to be returned. Not happy with it aesthetically. I think it should maybe be an EOF token. Refactor it later.

Reply:
#507